dwave/plugins/sklearn/transformers.py

```diff
@@ -293,7 +293,7 @@ def fit(
                 """
             )
 
-        sampleset = sampler.sample_cqm(cqm, time_limit=self.time_limit)
+        sampleset = sampler.sample_cqm(cqm, time_limit=self.time_limit, label="feature-selection")
```

Comment:
```suggestion
        sampleset = sampler.sample_cqm(cqm, time_limit=self.time_limit, label="dwave-scikit-learn-plugin Feature Selection")
```

Our label convention doesn't include this type of repo (not dwave-example, JN, or demo), the closest we have is [VirtualGraph flux bias measurements](https://github.com/dwavesystems/dwave-system/blob/bedfe5143a8579348be07e4ef5e8fe0646ce81ff/dwave/system/flux_bias_offsets.py), but this seems more consistent, especially when we have multiple FS repos.